functional-tests/functional/input/lime/Enums.lime

```diff
@@ -62,3 +62,9 @@ enum EnumWithDeprecatedItems {
     bar,
     baz
 }
+
+@Cpp(ToString)
```

Comment:
Since function is header only it doesn't make sense to introduce new attribute. Or move implementation to cpp.

Reply:
I've seen your comment in the ticket that compiler will leave out the function if not used and probably should defend my implementation on my own - but this is Daniel's decision.

Reply:
As we have C++ public APIs now, this needs to be guarded by an attribute. We are not going to pollute the whole public API with this new function all of a sudden.